src/Widgets/UserBox.vala

```diff
@@ -32,10 +31,8 @@ public class Session.Widgets.Userbox : Gtk.ListBoxRow {
         Object (user: user);
     }
 
-    public Userbox.from_data (string fullname, bool logged_in, bool is_guest = false) {
-        Object (fullname: fullname,
-                is_guest: is_guest,
-                user: null);
+    public Userbox.from_guest () {
```

Comment:
Make it even simple
```suggestion
    public Userbox.guest () {
```

Reply:
I think `from_guest` still feels more semantically correct to me :man_shrugging:

Reply:
I won't block on that but we usually have `Foo.from_bar ()` when it takes a Bar parameter